src/style.css

```diff
@@ -0,0 +1,96 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format(ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format(ttf);
+  font-style: bold;
+  font-weight: 300;
```

Comment:
`font-style` should be normal, there isn't such value as bold, check [here](https://www.w3schools.com/cssref/pr_font_font-style.php). set `font-weight: bold/800` instead